src/database/migration/file/migration_version_8.ts

```diff
@@ -0,0 +1,26 @@
+import {SQLiteDatabase} from 'react-native-sqlite-storage';
+
+import ChatSchema from '../../schema/ChatSchema';
+import Migration from './migration.types';
+
+const TABLE_NAME = ChatSchema.getTableName();
+
+class MigrationVersion8 implements Migration {
+  up = async (db: SQLiteDatabase): Promise<void> => {
+    const upQuery = `ALTER TABLE ${TABLE_NAME}
+      ADD COLUMN attachment_json TEXT DEFAULT NULL;`;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The SQL query for adding a new column to the table is correct. However, it's recommended to handle potential errors that might occur during the execution of this query. Wrapping the `executeSql` call in a try-catch block would allow you to handle any exceptions and provide more informative error messages.

```diff
+    try {
        await db.executeSql(upQuery);
+    } catch (error) {
+        console.error(`Error while migrating ${TABLE_NAME}: `, error);
+    }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->